client/src/App.css

```diff
@@ -128,6 +145,91 @@ a:hover, a:visited {
   padding: 200px;
 }
 
-.card {
-  box-shadow: 2px 2px 4px lightgrey;
+.heroSection2 {
```

Comment:
for each of these heroes, you'll need to update class names here as well

Reply:
Done